be/src/olap/olap_server.cpp

```diff
@@ -129,6 +129,12 @@ Status StorageEngine::start_bg_threads() {
             &_unused_rowset_monitor_thread));
     LOG(INFO) << "unused rowset monitor thread started";
 
+    RETURN_IF_ERROR(Thread::create(
```

Comment:
There is no such thread before. Is it special for cloud mode?

Reply:
not special, it's for both local and cloud mode to clean expired quering rowsets